packages/core/src/components/menu/menuItem.tsx

```diff
@@ -70,6 +70,14 @@ export interface IMenuItemProps extends IActionProps, ILinkProps {
      * @default true
      */
     shouldDismissPopover?: boolean;
+
+    /**
+     * Name of the HTML tag that wraps the MenuItem.
+     *
+     * By default a `<a>` is used
```

Comment:
remove this line, it's covered by `@default` below.

Reply:
Will do